src/gateway/admin-api/db-encryption.md

```diff
@@ -112,6 +112,40 @@ HTTP 201 Created
 HTTP 201 Created
 ```
 
+## Recovery Keyring from Database
+
+*This endpoint is only available with the `cluster` keyring strategy.*
```

Comment:
```suggestion
This endpoint is only available with the `cluster` keyring strategy.
```

Reply:
This was to be aligned with the original text https://github.com/Kong/docs.konghq.com/blob/bcafb595f4b59b4ce951b1d92325a200b3809769/src/gateway/admin-api/db-encryption.md?plain=1#L46-L50